robot-server/robot_server/service/legacy/models/settings.py

```diff
@@ -239,3 +239,37 @@ def validate_fields(
                 else:
                     raise ValueError(f"{key} is not a valid field or quirk name")
         return v
+
+
+class CameraEnable(BaseModel):
+    """Configuration value for enabling and disabling the Opentrons Camera or Live Stream."""
```

Comment:
```suggestion
    """Configuration value for Opentrons Camera and Live Stream enablement. Disabling the Camera also disables the Live Stream. Enabling the Camera retains the existing Live Stream enablement setting. """
```
Nit, since this toggle should control both the camera and livestream. Feel free to reword.